src/Fire.js

```diff
@@ -0,0 +1,53 @@
+import firebase from "firebase";
+
+class Fire {
+  constructor() {
+    this.init();
+    this.observeAuth();
+  }
+
+  init = () => {
+    if (!firebase.apps.length) {
+      firebase.initializeApp({
+        apiKey: "AIzaSyAmBNrwmBFVnA1fbey5MqrW83Qzkb6m-b8",
```

Comment:
We are not allowed to keep API keys/token on GitHub. Create .env and store data there.